<!DOCTYPE html>
<html lang="en">
<head>
	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
	<!-- Increase compatibility with older browsers that do not support HTML5 -->
<<<<<<< HEAD
	<script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
=======
	<!--<script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.min.js"></script>-->
>>>>>>> SangHo-V1
	<!-- Load jQuery from the web, if down then local -->
	<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
	<script type="text/javascript">
		if (typeof jQuery == 'undefined') {
			document.write(unescape("%3Cscript src='js/jquery-1.10.2.min.js' type='text/javascript'%3E%3C/script%3E"));
		}
	</script>
	<!-- Load Google Web Font -->
	<link href='https://fonts.googleapis.com/css?family=Raleway:400,200' rel='stylesheet' type='text/css' />
	<!-- Load CSS for the page -->	
	<link href='css/hoksok.css' rel='stylesheet' type='text/css' />
	<link href="css/hover.css" rel="stylesheet" media="all">
	<!-- Title of the web page - as displayed in the tab -->
	<title>UMaryland iGEM 2015 - Hok/Sok</title>
</head>
<body>
	<div id="div-topbar"> 		<!-- Start of top header navigation bar 		-->
		<nav>
			<a href="index.html">
			<img src="img/logo.png" id="img-logo">
			</a>
			<a href="http://2016.igem.org/Main_Page">
			<img src="img/igemLogo.png" id="img-igemlogo">
			</a>
			<ul>
				<li><a href="projects.html" class="hvr-float">Projects</a></li>
				<li><a href="parts.html" class="hvr-float">Parts</a></li>
				<li><a href="modeling.html" class="hvr-float">Modeling</a></li>
				<li><a href="outreach.html" class="hvr-float">Outreach</a></li>
				<li><a href="notebook.html" class="hvr-float">Notebook</a></li>
				<li><a href="about.html" class="hvr-float">About Us</a></li>
			</ul>	
		</nav>
	</div>	
	<script>
		<!-- For nav bar scrolling + hovering -->
		$('nav').mouseover(
			function () {
				$('#img-logo').css('transition', '.35s ease-in');
				$('#img-logo').css('opacity', '1');
				$('#img-igemlogo').css('transition', '.35s ease-in');
				$('#img-igemlogo').css('opacity', '1');
				$('nav ul').addClass("hover");
				$('nav').css('opacity', '1');
				$('nav').css('transition', '.5s ease');
			}	
		),
		$('nav').mouseout(
			function () {
			var scroll = $(window).scrollTop();
				if (scroll > 0) {
					$('#img-logo').css('transition', '.25s ease');
					$('#img-logo').css('opacity', '0');
					$('#img-igemlogo').css('transition', '.25s ease');
					$('#img-igemlogo').css('opacity', '0');
					$('nav ul').removeClass("hover");
					$('nav').css('opacity', '.5');
					$('nav').css('transition', '.5s ease');
				}
			}
		);
		$(function() {
			var nav = $('nav ul');
			var navList = $('nav ul li');
			$(window).scroll(function() {
				var scroll = $(window).scrollTop();
				if (scroll > 0) {
					nav.addClass("scrolled");
					nav.removeClass("hover");
					$('#img-logo').css('transition', '.25s ease');
					$('#img-logo').css('opacity', '0');
					$('#img-igemlogo').css('transition', '.25s ease');
					$('#img-igemlogo').css('opacity', '0');
					$('nav').css('transition', '.5s ease');
					$('nav').css('opacity', '.5');
				} else {
					nav.removeClass("scrolled");
					$('nav').css('transition', '.5s ease');
					$('nav').css('opacity', '1');
					$('#img-logo').css('transition', '.35s ease-in');
					$('#img-logo').css('opacity', '1');
					$('#img-igemlogo').css('transition', '.35s ease-in');
					$('#img-igemlogo').css('opacity', '1');
					
				}
			});
		});
	</script>
	<div id="div-text"> <!-- Start of main text of page -->
		<!-- Heading text -->
		<div id="div-pic"> 
		<!-- where background image below titles show -->
			<h1>Hok/Sok</h1>
			<h2>Antibiotic Independent Plasmid Retention</h2>
			<h3>A natural internal maintenance cassette relying on internal mRNA silencing that is an alternative to common antibiotic-based methods without relying on exogenous drugs</h3>
		</div>
		<table id="table-selector"> <!-- Table of navigational links -->
			<thead>
				<!-- .border classes signify where border needs to be drawn -->
				<!-- navigator class trigger javascript on that element     -->
				<!-- data-select attribute sends to script which element    -->
				<!-- was clicked 											-->
				<th class="borderRight navigator borderTop" id="th-purpose" data-select="purpose"><p>Purpose</p></th>
				<th class="borderBoth navigator borderTop" id="th-background" data-select="background"><p>Background</p></th>
				<th class="borderBoth navigator borderTop" id="th-experiment" data-select="experiment"><p>Experiment</p></th>
				<th class="borderLeft navigator borderTop" id="th-results" data-select="results"><p>Results</p></th>
			</thead>
		</table>
		<ul id="ul-experiment" class="subNavList">
				<li class="subNav" id="th-construct" data-select="construct">Construct</li>
				<li class="subNav" id="th-plating" data-select="plating">Plating</li>
				<li class="subNav" id="th-growth" data-select="growth">Growth Curve</li>
				<li class="subNav" id="th-fluorescence" data-select="fluorescence">Fluorescence</li>
				<li class="subNav" id="th-loss" data-select="loss">Fluorescence Loss</li>
				<li class="subNav" id="th-sequence" data-select="sequence">Sequence</li>
		</ul>
		<ul id="ul-results" class="subNavList">
				<li class="subNav" data-select="plating2">Plating</li>
				<li class="subNav" data-select="growth2">Growth Curve</li>
				<li class="subNav" data-select="fluorescence2">Fluorescence</li>
				<li class="subNav" data-select="loss2">Fluorescence Loss</li>
				<li class="subNav" data-select="sequence2">Sequence</li>
				<li class="subNav" data-select="conclusions">Conclusions</li>
				<li class="subNav" data-select="considerations">Considerations</li>
		</ul>
		<!-- .blockText makes text initially hidden until clicked upon in navigational element 	   -->
		<!-- start the div shown initially and when 'purpose' nav element is clicked 			   -->
		<div id="div-purpose" class="blockText initialDisplay"> 
			<div class="longText"> <!-- .longText reduces width of text and centers slightly left -->
								   <!-- formats font, line height, font weight					  -->
				<p>The ability to use plasmids as vectors to introduce genes of interest in E. coli is one of the most essential bioengineering tools. However, one of the limitations of transforming a bacterium with a plasmid, is that the organism will eventually eject the plasmid over time. To counter this, scientists add a positive selective pressure on E. coli to retain plasmids carrying resistance genes through the use of antibiotics.</p> 
				<p>While this technique has proven to be reliable and effective, there are many limitations. The prevalent use of antibiotics both for medical and agricultural purposes has rapidly increased the number of pathogens that harbor antibiotic resistant genes. As a result, there is a pressing need to find an alternative to antibiotic use for plasmid maintenance to prevent the spread of antibiotic resistant genes.</p>
			</div>
			<!-- .imageEmbed creates border around picture to improve transition to background 	-->
			<!-- increase the space above and below the image									-->
			<!-- make sure .imageEmbed is outside of div.longText								-->
			<img class="imageEmbed imgW600" id="img-antibiotic" src="img/antibiotic.jpg" alt="A picture of antibiotics"/>
			<div class="longText">
				<p>Many synthetic biology projects which focus of solving health or environmental issues are confined to the lab because of these limitations. The University of Maryland iGEM team seeks to solve this problem by developing an alternative plasmid maintenance system that should liberate other iGEM teams from the dependance on antibiotic usage.</p> 
				<p>We hypothesized that Hok-Sok, our plasmid maintenance system, could maintain recombinant plasmids, as it does natural ones. We also hypothesized that Hok-Sok would have a slight negative effect on bacterial growth rate, in line with other alternative maintenance systems such as sRNBC (K817015), as well as the amount of protein expression due to competing parallel promoters. In order to answer these questions, we set up a variety of experimental tests.</p>

			</div>
		</div>
		<!-- start the div shown initially and when 'background' nav element is clicked -->
		<div id="div-background" class="blockText">
			<div class="longText">
				<p>The Hok-Sok cassette contains three genes:</p>
				<ol>
				<li><em>hok</em> - host-killing</li>
				<li><em>sok</em> - suppressor of killing</li>
				<li><em>mok</em> - modulator of killing</li>
				</ol>
				<p><em>Hok</em>, located on the forward strand, encodes a 52 amino acid toxin that is capable of permeating the cell membrane, leading to a loss of electrochemical potential and cell death. The hok toxin acts with "single-hit" kinetics, indicating that a single molecule of translated <em>hok</em> is sufficient for killing a cell. Hok is freely transcribed under a weak promoter.</p> 
				<p>In order to stay alive, cells need to be able to block translation of the hok transcript. <em>Mok</em>, located on the same locus as <em>hok</em>, modulates <em>hok</em> translation by the ribosome. <em>Mok</em> is located on a different reading frame than <em>hok</em> and the two genes are always co-transcribed as part of a single mRNA transcript.</p>
				<p><em>Mok</em> is then recognized by <em>sok</em>, a short ssRNA "antitoxin" transcribed under a strong constitutive promoter on the reverse strand of the Hok-Sok cassette. <em>Sok</em> is capable of forming a double-stranded RNA duplex near the 5'-end of the <em>mok</em> transcript, indirectly blocking the <em>hok</em> open reading frame and preventing the transcript from being read. The dsRNA duplex is then recognized by RNAse III, leading to enzymatic degradation of the silenced transcript.</p>
			</div>
			<img class="imageEmbed imgW800" src="img/hoksokdiagram.png" alt="A diagram explaining how Hok/Sok promotes plasmid retention" />
			<div class="longText">
				<p>This cassette promotes plasmid maintenance because the <em>hok</em> mRNA, due to a high degree of secondary structure, has a long half-life, measured at 20 minutes, or approximately the time of one growth cycle. <em>Sok</em>, on the other hand, has a half-life of only 30 seconds. Thus in order to continue suppressing <em>hok</em> translation, the cell must retain the plasmid and the <em>sok</em> gene through cell division.</p> 
				<p>If the plasmid is "lost", or not passed down to the daughter cell, then both the <em>hok</em> and <em>sok</em> coding regions will be lost. However, due to its long half-life, previously transcribed <em>hok</em> mRNA will remain in the cell while previously transcribed <em>sok</em> rapidly degrades. No longer silenced, <em>hok</em> mRNA will be then translated, killing the daughter cell that did not maintain the plasmid. In nature, this is how the Hok-Sok system maintains antibiotic resistance in E. coli in the absence of antibiotic pressure.</p>
			</div>
		</div>
		<!-- start the div shown initially and when 'experiment' nav element is clicked -->
		<div id="div-experiment" class="blockText">
			<div class="longText">
				<h4 class="firstHeader">Hok/Sok Construct</h4> <!-- h4 mini heading within the text, formatted specifically within .longText -->
				<p>Prior to experimentation, we had to insert the Hok-Sok construct into pSB1C3 in order to make it a BioBrick. We originally planned to PCR amplify the cassette out of the R1 plasmid of E. coli, but we were unable to find an suitable wild-type strain that was easily available. Instead, we turned to synthesizing the construct as a gBlock from IDT. As a 580 bp dsDNA fragment, it was suitable for addition via Gibson Assembly into pSB1C3.</p>
			</div>
			<img class="imageEmbed imgW800 noBottomMargin" src="img/construct.png" alt="An image of our BioBrick" />
			<div class="longText">
				<h4 id="h4-plating">Plating Studies</h4>
				<p>While our fluorescence studies were effective at measuring protein expression over time, we wanted a second test that would more directly measure whether or not plasmids were being maintained throughout generations. Using the identical cultures as the fluorescence tests, we devised a plating protocol involving a challenge of chloramphenicol every 24 hours.</p>
				<ol>
					<li>Dilute each culture (1 : 106) with LB media</li>
					<li>Plate 10 µL of each diluted culture on a LB-agar + chloramphenicol plate</li>
					<li>Incubate plates overnight</li>
					<li>Count colonies the next day</li>
				</ol>
				<p>The goal in doing this was to determine how many bacteria were surviving by retaining their plasmids. We did not discriminate between the color of colonies.</p>
				<p>For continuing generations of BL21 strain E. coli, we observed that on the plates for groups A and B, there was growth but no redness. If the bacteria were retaining the plasmids with the chloramphenicol resistance, the RFP gene should have been expressed and the colonies should fluoresce. We hypothesized that the chloramphenicol resistance gene was being recombined into the bacterial genome so the bacteria could therefore freely eject our inserted plasmids. As BL21 carries the gene for recombinase, it is possible. However, DH5α, as a common cloning strain, does not have recombinase. We created a new generation with every group (A, B, C, D, and E) to test whether the same plate would have similar results or once the bacteria stopped fluorescing, there would be no growth on the plates.</p>
			</div>
			<img class="imageEmbed imgW600 noBottomMargin" src="img/RFP.png" alt="Bacteria expressing RFP" />
			<div class="longText">
				<h4 id="h4-growth">Growth Curve</h4>
				<p>While the Hok-Sok cassette may help to maintain a plasmid, it may also impact the rate of cell growth. We were unsure of the level of stress that came from additional hok and sok translation, and we decided to measure whether or not cultures containing the Hok-Sok construct would grow at a similar rate to control. We created a growth curve of Hok/Sok in comparison to controls to test the effectiveness of the Hok/Sok system in keeping the bacteria alive. We had four groups:</p>
				<ul>
					<li>Hok/Sok without chloramphenicol</li>
					<li>Hok/Sok with chloramphenicol</li>
					<li>RFP without chloramphenicol</li>
					<li>RFP with chloramphenicol</li>
				</ul>
				<p>We started growing 250 mL cultures and monitored the OD at 600nm using a spectrophotometer over the span of 7.5 hours.</p>
			</div>
			<div class="longText">
				<h4 id="h4-fluorescence">Fluorescence Studies</h4>
				<p>In order to determine if Hok/Sok was capable of maintaining a plasmid without antibiotic pressure, we decided to use a visual reporter gene to quantify the ability of Hok/Sok to maintain plasmids over many generations. We decided to use RFP along with a degradation tag as the reporter gene. The most suitable candidate was an unstable LVA-tagged RFP that has a half-life of 1 hour. The shorter half-life allows for more frequent measurements of protein production that would not aggregate over time.</p> 
				<p>We combined a constitutive promoter and RBS to the LVA-tagged RFP through 3A assembly. We transformed this construct to E. coli DH5a to confirm the effectiveness of this reporter gene and its expression through increased fluorescence. Afterwords we ordered a g-block of our Hok/Sok+reporter construct. The expression of this reporter gene is proportional to plasmid number. Therefore, we concluded that if the cells containing a plasmid with both Hok/Sok and reporter gene could maintain fluorescence over many generations without the positive pressure of antibiotics compared to our controls, Hok/Sok can be used as a viable plasmid maintenance system.</p>
				<p>We used two E. coli strains for our testing. Originally, we used BL21 strain of E. coli because it is known to be the best for testing because the cell lacks proteases; the protein expression is optimal because the proteins are not digested by the enzymes. After testing BL21, we transitioned to the DH5a strain of E. coli because the cells lack recombinase.</p>
				<h4 id="h4-loss">Fluorescence Loss Analysis</h4>
				<p>Interested as to why our cells were losing fluorescence in the span of a week, we increased the level of protein expression in order to observe this effect on a larger scale. This was done by switching cell lines from DH5α to BL21, which is optimized for protein expression due to the removal of several proteases. We repeated plating experiments in triplicate in order to determine if fluorescence loss would be as dramatic in such a small span of time. Plates were exposed to UV light using a transilluminator in order to visually observe fluorescence loss over many generations.</p>
				<p>In addition to our observations, we also wanted to make sure that our cultures had not been contaminated with a non-fluorescent, chloramphenicol resistant bacteria that had out-competed our intended culture. We thus performed a gram stain in order to verify that our bacteria were gram negative bacilli.</p>
				<h4 id="h4-sequence">Sequence Analysis</h4>
				<p>In order to find a definite answer as to the loss of fluorescence, we took minipreps of our non-fluorescent DH5α cultures, digested them to extract their inserts, and separated them using agarose gel electrophoresis in order to determine whether or not they were the proper size. For samples that contained an insert of the proper size, the miniprepped plasmid was then sequenced in order to determine the genetic reason as to the fluorescence loss.</p>
			</div>
		</div>
		<!-- start the div shown initially and when 'results' nav element is clicked -->
		<div id="div-results" class="blockText">
			<div class="longText">
				<h4 id="h4-plating2">Plating Tests</h4>
				<p>Over time, all cultures appeared to maintain antibiotic resistance, except for the negative control, which experienced no pressure during its growth. The following figures demonstrate the presence of resistant colonies over many hundreds of bacterial generations. Most importantly, the Hok-Sok culture grown without antibiotic was shown to maintain chloramphenicol resistance. This sets up one of our major conclusions: the Hok-Sok cassette is capable of maintaining a recombinant plasmid for an extended period of time.</p>
			</div>
				<!-- table that formats the images and captions 								-->
				<!-- h5 and p formatted differently, specific to elements within table element 	-->
				<table id="table-plating">
					<tbody>
					<tr>
						<td>
							<!-- h5 - heading above image -->
							<h5>Negative Control (No Pressure)</h5> 
							<!-- .figure fixes size of the pictures -->
							<img src="img/fig1.png" class="figure" alt="Figure showing the number of colonies when under no selective pressure" />
							<!-- p - caption below image -->
							<p><strong>Figure 1.</strong> Negative control was K1783002 (constitutive unstable RFP) grown in media without chloramphenicol. After 2-3 days, plates demonstrate fewer than 10 colonies per plate, suggesting that antibiotic resistance is now only minimally present in the culture.</p>
						</td>
						<td>
							<h5>Chloramphenicol Pressure Only</h5>
							<img src="img/fig2.png" class="figure" alt="Figure showing the number of colonies when under chloramphenicol pressure only"/>
							<p><strong>Figure 2.</strong> K1783002 (constitutive unstable RFP) grown in media with chloramphenicol. Colonies persist through several days of plating, suggesting that antibiotic resistance was maintained throughout the test.</p>
						</td>
					</tr>
					<tr>
						<td>
							<h5>Chloramphenicol + Hok-Sok Pressure</h5>
							<img src="img/fig3.png" class="figure" alt="Figure showing the number of colonies when under chloramphenicol and hok-sok pressure"/>
							<p><strong>Figure 3.</strong> K1783003 (constitutive unstable RFP under Hok-Sok regulation) grown in media with chloramphenicol. Colonies persist through several days of plating, suggesting that antibiotic resistance was maintained throughout the test.</p>
						</td>
						<td>
							<h5>Hok-Sok Pressure Only</h5>
							<img src="img/fig4.png" class="figure" alt="Figure showing the number of colonies when under hok-sok pressure only"/>
							<p><strong>Figure 4.</strong> K1783003 (constitutive unstable RFP under Hok-Sok regulation) grown in media without chloramphenicol. Colonies persist through several days of plating, suggesting that antibiotic resistance was maintained throughout the test.</p>
						</td>
					</tr>
				</table> <!-- end table of results -->
			<div class="longText">
				<h4 id="h4-growth2">Growth Curve</h4>
				<p><strong>Figure 5. </strong>Growth curves of parallel cultures grown with and without chloramphenicol. The presence of Hok-Sok on the inserted plasmid does not appear to have a major effect on the bacterial growth rate. This is in line with our qualitative observations, where we do not observe any major difference in cell density during plating or fluorescence studies.</p>
			</div>
			<img class="imageEmbed imgW600 noBottomMargin" src="img/figgrowth.png" alt="Figure showing growth rates using differnet maintenance systems" />
			<div class="longText">
				<h4 id="h4-fluorescence2">Fluorescence Studies</h4>
				<p>Our fluorescence studies supported the findings of our plating tests. We were able to observe that fluorescence was rapidly lost in the negative control. This was expected due to plating tests demonstrating the loss of plasmid from that group. We were pleasantly surprised to observe that Hok-Sok was able to maintain fluorescence for a longer period of time than typical chloramphenicol pressure.</p>
			</div>
			<!-- table that formats the images and captions 								-->
			<!-- h5 and p formatted differently, specific to elements within table element 	-->
			<table id="table-fluro">
				<tbody>
					<tr>
						<!-- .centerText centers the heading of the subsection of the table  -->
						<td colspan="2"><h5 class="centerText" >DH5α Cell Line</h5></td>
					</tr>
					<tr>
						<td>
							<h5>Chloramphenicol Pressure</h5>
							<img src="img/fig6.png" class="figure" />
							<p><strong>Figure 6.</strong> Fluorescence measurements of K1783002 (Constitutive unstable RFP) in DH5α cells grown in media containing antibiotic. All measurements are blank-subtracted. Fluorescence is initially high, but rapidly diminishes over time.</p>
						</td>
						<td>
							<h5>No Pressure</h5>
							<img src="img/fig7.png" class="figure" />
							<p><strong>Figure 7.</strong> K1783002 (Constitutive unstable RFP) in DH5α cells grown in media without antibiotic. Fluorescence is initially low and rapidly diminishes over time. This result supports the notion that cells grown without any pressure quickly lose their plasmids.</p>
						</td>
					</tr>
					<tr>
						<td>
							<h5>Chloramphenicol + Hok/Sok Pressure</h5>
							<img src="img/fig8.png" class="figure" />
							<p><strong>Figure 8.</strong> K1783003 (Hok-Sok+Constitutive unstable RFP) in DH5α cells grown in media with antibiotic. Fluorescence is initially moderate and diminishes over time. The use of two maintenance systems does not appear to better maintain fluorescence levels than chloramphenicol alone. We suggest that the proximity of the Hok-Sok cassette to the RFP construct somewhat represses expression of RFP due to promoter interference, leading to a lower initial reading.</p>
						</td>
						<td>
							<h5>Hok/Sok Pressure</h5>
							<img src="img/fig9.png" class="figure" />
							<p><strong>Figure 9.</strong> K1783003 (Hok-Sok+Constitutive unstable RFP) in DH5α cells grown in media without antibiotic. Fluorescence is initially moderate, but remains relatively constant over time, a one-time spike notwithstanding.</p>
						</td>
					</tr>
					<tr>
						<td colspan="2"><h5 class="centerText" >BL21 Cell Line</h5></td>
					</tr>
					<tr>
						<td>
							<h5>Chloramphenicol Pressure</h5>
							<img src="img/fig10.png" class="figure" />
							<p><strong>Figure 10.</strong> Fluorescence measurements of K1783002 (Constitutive unstable RFP) in BL21 cells grown in media containing antibiotic. All measurements are blank-subtracted. Fluorescence is variable throughout testing suggesting an uncorrected factor influencing results. Note: Our initial tests using the BL21 cell line were inconclusive due the need to calibrate our testing protocol.</p>
						</td>
						<td>
							<h5>No Pressure</h5>
							<img src="img/fig11.png" class="figure" />
							<p><strong>Figure 11.</strong> Fluorescence measurements of K1783002 (Constitutive unstable RFP) in BL21 cells grown in media without antibiotic.</p>
						</td>
					</tr>
					<tr>
						<td>
							<h5>Chloramphenicol + Hok/Sok Pressure</h5>
							<img src="img/fig12.png" class="figure" />
							<p><strong>Figure 12.</strong> Fluorescence measurements of K1783003 (Hok-Sok+Constitutive unstable RFP) in BL21 cells grown in media with antibiotic.</p>
						</td>
						<td>
							<h5>Hok/Sok Pressure</h5>
							<img src="img/fig13.png" class="figure" />
							<p><strong>Figure 13.</strong> Fluorescence measurements of K1783003 (Hok-Sok+Constitutive unstable RFP) in BL21 cells grown in media without antibiotic.</p>
						</td>
					</tr>
			</table> <!-- end table of fluroescence results -->
			<div class="longText">
				<h4 id="h4-loss2">Analysis of Fluorescence Loss</h4>
			</div>
			<table id="table-loss">
				<tbody>
					<tr>
						<td>
							<img src="img/14.jpg" class="figure" />
							<p><strong>Figure 14.</strong> Over time, colonies appear to stop producing RFP.</p>
						</td>
						<td>
							<img src="img/15.jpg" class="figure" />
							<p><strong>Figure 15.</strong> This effect occurs much more slowly when the Hok-Sok cassette is adjacent to the RFP construct.</p>
						</td>
					</tr>
					<tr>
						<td>
							<img src="img/16.jpg" class="figure" />
							<p><strong>Figure 16.</strong> This effect occurs even when the culture is under chloramphenicol pressure.</p>
						</td>
						<td>
							<img src="img/17.jpg" class="figure" />
							<p><strong>Figure 17.</strong> With or without added chloramphenicol, Hok-Sok appears to be able to maintain a relatively constant level of fluorescence..</p>
						</td>
					</tr>
				</tbody>
			</table>
			<div class="longText">
				<p>Representative gram stain of culture. Gram staining of our samples confirms that our samples are gram-negative bacilli, making it highly unlikely that our cultures have been contaminated.</p>
			</div>
			<img class="imageEmbed imgW300 noBottomMargin" src="img/gram.png" alt="Representative gram stain of culture." /> 
			<div class="longText">
				<h4 id="h4-sequence2">Sequence Analysis</h4>
				<p>Top right bands of gel are of negative control in DH5-alpha. They demonstrate that the cells no longer have the plasmid of interest. Gels generally show that plasmids are kept whenever a form of pressure is placed on the cell. Why then, is RFP not being expressed? Our sequencing results showed random mutations in the promoter and coding region of the RFP construct.</p> 
				<p>This is a valuable lesson that, with any BioBrick construct, mutations and evolution is inevitable. However, plasmids that were maintained with Hok-Sok alone (no chloramphenicol) did not display mutations in the RFP construct. The large difference in protein expression over multiple days, as shown by our fluorescence and plating tests, suggests to us that the presence of Hok-Sok, combined with the absence of chloramphenicol pressure, is putting a smaller evolutionary pressure on the bacterium.</p>
			</div>
			<img class="imageEmbed imgW600 noBottomMargin" src="img/sequence.jpeg" /> 
			<div class="longText">
				<h4 id="h4-conclusions">Conclusions and Future Plans</h4>
				<p>Through our plating tests and fluorescence measurements, we were able to successfully observe the plasmid maintenance ability of the Hok-Sok system. In comparison to a negative control grown without chloramphenicol pressure, cells containing Hok-Sok were able to maintain their antibiotic resistance for a significant period of time, as shown from our plating tests.</p> 
				<p>In addition, we noted the ability of Hok-Sok had a negative effect on protein production, as shown from our fluorescence study. However, this level of production was consistent over a long period of time, contrasting with traditional pressure systems where mutations were able to build up quickly, shutting down production of RFP, which has no biological usefulness for the cell. </p>
				<p>We thus suggest that the downregulation of RFP production via Hok-Sok is capable of decreasing the evolutionary pressure against the removal on non-essential genes. This effect will be further studied in later experiments, where the distance between the hok-sok cassette and a gene of interest is lengthened. Overall, we suggest use of the Hok-Sok system as an effective method for internal plasmid maintenance without the use of antibiotics.</p>
				<h4 id="h4-considerations">Considerations</h4>
				<p><strong>PCR of Hok-Sok Construct</strong><br />
				Amplification of the Hok-Sok cassette is difficult due to the high inherent secondary structure in the construct. Hok ssDNA is capable of naturally folding into a stable secondary structure, resulting in early terminations and other side products. We recommend that a reaction buffer suitable for difficult templates be used, such as Phusion GC buffer + added DMSO.</p>
				<p><strong>Measuring OD of RFP Cultures</strong><br />
				Due to the close proximity of the emission wavelength of RFP (584 nm) and the classical absorbance wavelength for measuring cell density (600 nm), it is difficult to accurately determine the cell density of cultures that are expressing RFP. Given more time to calibrate our testing measurements, we would either have used an alternative wavelength for measuring OD (>600 nm), used a hemocytometer as an alternate counting method, or switched to GFP as an alternative fluorescent marker whose emission wavelength differs from 600 nm by a greater amount.</p>
				<p><strong>Moving Forward</strong><br />
				In the future there is great potential to use the Hok-Sok system both in vitro and in vivo. In the laboratory, using plasmids which contain the Hok-Sok cassette along with some sort of positive selection marker, such as a fluorescent protein, instead of an antibiotic resistance gene would permit for positive selection of desired colonies without the use of antibiotics.</p>
				<p>Furthermore, the Hok-Sok system could have a transformative role outside the lab. Synthetic biology projects often rely upon genetically modified bacteria which are designed for release into the soil, water sources, and ingestion by animals. Our 2014 project, for example, involved E. coli that could one day detect an oyster pathogen, and the University of Toulouse 2014 iGEM team was interested in designing bacteria capable of defending trees against fungal attack.</p> 
			</div>
			<img class="imageEmbed imgW600" src="img/oyster.jpg" alt="A picture of oysters"/>
			<div class="longText">
				<p>If these bacteria carried antibiotic resistance genes, there would be ample concern about other bacteria acquiring this resistance through horizontal gene transfer. However, if the Hok-Sok system was used to maintain plasmids, it would reduce concern over increasing the number of antibiotic resistant bacterial strains.</p>
				<p>In addition, there is potential to take advantage of the Hok-Sok cassette as a method of post-transcriptional control over a variety of genes. As hok is indirectly regulated by the upstream binding of sok to mok, there is potential to control protein expression through using the interplay of sok and mok to prevent the translation of various mRNA transcripts. Although we did not focus on this, there is great potential for experimentation.</p>
				<p>As mentioned earlier, many synthetic biology projects, including those within iGEM, intend to release their final product into nature. As our testing supports the conclusion that Hok-Sok can successfully be implemented to maintain plasmids without antibiotics, it would be intriguing to test this further ourselves or see other iGEM teams test this by implementing the Hok-Sok cassette as the maintenance system on a project where the final product is intended for release into nature. Successful implementation of this would further prove the efficacy of Hok-Sok as an alternative plasmid maintenance system and as a method for combating the widespread use of antibiotics and growing population of antibiotic resistant bacteria.</p>
			</div>
		</div>
		<!-- end of #div-results -->
	</div>
	<div id="bot-nav">
		<ul>
			<li> <a href="https://www.facebook.com/UMarylandiGEM/">Facebook</a></li>
			<li>|</li>
			<li> <a href="https://www.instagram.com/umarylandigem/">Instagram</a></li>
			<li>|</li>	
			<li> <a href="mailto:umarylandigem@gmail.com">Email</a></li>
		</ul>
	</div>
</body>
<script>
var clicked = '';
var current = 'purpose'; // sets initial display to be purpose
 $('.navigator').click( function() { 					// when any element with .class is clicked, trigger function
	var selected = $(this).attr('data-select');			// grabs the name of the navigational element clicked
	if (selected != current) {							// only change display / trigger animation if clicking different nav
		changeText(selected)
	}
 })
 
var dir = location.hash;								// grabs the anchor (text following the #)
var n = dir.indexOf('?');								// and initially sets the page 
dir = dir.substring(0, n != -1 ? n : dir.length);		
dir = dir.replace('#','');
if (dir == '') {
	dir = 'purpose';
}
changeText(dir)
switch (dir) {
	case 'purpose':
        break;
    case 'background':
        break;
	case 'experiment':
        displayTable(dir)
        break;
	case 'results':
        displayTable(dir)
        break;
    default:
        break;
}

$(".subNav").click(function() {
	clicked = $(this).attr('data-select');	
    $('html, body').animate({
        scrollTop: $("#h4-"+ clicked).offset().top
    }, 0000);
});

function displayTable(selected) {
	$('#ul-'+selected).css('display','block');
}

function changeText(selected) {
	$('.navigator').css('border-bottom','1px solid #A8A8A8');		// make all navigational element have no bottom border
	$('#th-' + selected).css('border-bottom', '4px solid #A8A8A8');	// the clicked navigational element given border		
	$('.blockText').css('display','none');
	$('.subNavList').css('display','none');							// hides all text
	$('.blockText').css('margin-left', '60px')						// slighly shifts right all text to set up for animation
	$('#div-' + selected).css('display','block');
	displayTable(selected)											// the div containing the text is displayed
	$('#div-' + selected).animate({ marginLeft: '0px'}, 200);		// the text is animated to quickly move left
	current = selected;												// sets the current display
	location.hash = current;
}

 
</script>
</html>